server/conn.go

```diff
@@ -249,12 +250,12 @@ type handshakeResponse41 struct {
 }
 
 // parseOldHandshakeResponseHeader parses the old version handshake header HandshakeResponse320
-func parseOldHandshakeResponseHeader(packet *handshakeResponse41, data []byte) (parsedBytes int, err error) {
+func parseOldHandshakeResponseHeader(ctx context.Context, packet *handshakeResponse41, data []byte) (parsedBytes int, err error) {
 	// Ensure there are enough data to read:
 	// https://dev.mysql.com/doc/internals/en/connection-phase-packets.html#packet-Protocol::HandshakeResponse320
-	log.Debugf("Try to parse hanshake response as Protocol::HandshakeResponse320 , packet data: %v", data)
+	logutil.Logger(ctx).Debug("try to parse hanshake response as Protocol::HandshakeResponse320", zap.ByteString("packet data", data))
 	if len(data) < 2+3 {
-		log.Errorf("Got malformed handshake response, packet data: %v", data)
+		logutil.Logger(ctx).Error("got malformed handshake response", zap.ByteString("packet data", data))
```

Comment:
Should we use `packetData`?

Reply:
i perfer `data`, zz-jason advise this, ok I will take `packetData`